library/Haste/Model/Relations.php

```diff
@@ -84,6 +91,12 @@ public function addRelationCallbacks($strTable)
                 $GLOBALS['TL_DCA'][$strTable]['fields'][$strField]['filter'] = false;
                 static::$arrFilterableFields[$strField] = $arrRelation;
             }
+
+            // Use custom serach filtering
```

Comment:
typo

Reply:
Thx, fixed in https://github.com/codefog/contao-haste/pull/121/commits/37bc1dd7fd5c0660a20e645076f084674c6c7819